src/jobs/deploy_env.yml

```diff
@@ -34,13 +34,39 @@ parameters:
   helm_additional_args:
     type: string
     default: ""
+  show_changelog:
+    type: boolean
+    default: false
+    description: When true, displays what is new since the previous deployment.
+  k8s_deployment_name:
+    type: string
+    default: PROJECT_NAME
+    description: The Deployment resource's name in Kubernetes to interrogate for the previous deployment's version
+  fixed_previous_version:
+    type: string
+    default: ""
+    description: Instead of reading from Kubernetes deployment, provide a fixed APP_VERSION for the previous deployment
```

Comment:
why would you choose to set this?

Reply:
with real apps, I wouldn't. this is only used here for (fake) integration testing

Reply:
happy to remove it, but then I don't know how to get a genuine `kubectl` response with a valid circleci-orb SHA

Reply:
quite happy for it to be there, just would be worth prefixing the `description` with _Only used for integration testing_ or similar so that it is clear that real apps shouldn't set it